recipes/udunits2/build.sh

```diff
@@ -0,0 +1,25 @@
+#!/bin/bash
+
+if [[ $(uname) != Darwin ]]; then
+    yum install -y automake libtool texinfo
```

Comment:
Not according to manylinux. That said, they are pretty fundamental for some builds... I'd say we need to separate core dependencies from build dependencies on our build OSes. In practice, that might mean we want a ```automake``` distribution which we can mark as a build dep of udunits2...